### PR TITLE
Souvenir updates

### PR DESCRIPTION
**Pull request by KingSlendy**
Souvenir support for:
-Planets
-Shapes And Bombs
-Wavetapping

### CODE REVIEW

File: HTML/Wavetapping.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿<!DOCTYPE html>
+﻿
+<!DOCTYPE html>
```
Comment (samfun123) [inline HTML/Wavetapping.html]: Why did you add a blank line?

Comment (KingSlendy) [reply HTML/Wavetapping.html]: What, I'm not sure how that appeared
